lib/model/QubitJob.php

```diff
@@ -117,21 +117,24 @@ public function getCompletionDateString()
    */
   public function getStatusString()
   {
+    $i18n = sfContext::getInstance()->i18n;
+    $unknown = $i18n->__('Unknown');
+
     if (!isset($this->statusId))
     {
-      return 'unknown';
+      return $unknown;
     }
```

Comment:
You're right! I guess this is the result of having switched to Python for a while now, where `AttributeError` would be raised if you try to access to a member of an object that is not defined. I'll fix it!